cocos/base/CCConsole.cpp

```diff
@@ -492,6 +492,7 @@ Console::Console()
 , _isIpv6Server(false)
 , _sendDebugStrings(false)
 , _bindAddress("")
+, _commandSeparator(DEFAULT_COMMAND_SEPARATOR)
```

Comment:
this is a mistake, double init same value

Reply:
It's not my code here is my code: https://github.com/gestern/cocos2d-x/blob/1e780cb1fe9962c24d9aff9acbd5480052d7ca47/cocos/base/CCConsole.cpp#L488

Reply:
we should search that codes on the branch `gestern:android-getsafearearect`, refer to:

https://github.com/gestern/cocos2d-x/blob/android-getsafearearect/cocos/base/CCConsole.cpp#L495